csharp/samples/getting-started-receiving.md

```diff
@@ -0,0 +1,201 @@
+# Get started with Event Hubs in .NET
+
+## What will be accomplished
+
+In this tutorial, we will write a console application to receive messages from an Event Hub using the **EventProcessorHost**.
+
+## Prerequisites
+
+1. [Visual Studio 2015](http://www.visualstudio.com).
+
+2. An Azure subscription.
+
+3. [An Event Hubs namespace]().
```

Comment:
I need to add a link to show how to provision namespaces/entities, so this will become a link